src/coreclr/jit/switchrecognition.cpp

```diff
@@ -89,7 +91,15 @@ bool IsConstantTestCondBlock(const BasicBlock* block,
             if ((op1->IsCnsIntOrI() && !op1->IsIconHandle()) ^ (op2->IsCnsIntOrI() && !op2->IsIconHandle()))
             {
                 // TODO: relax this to support any side-effect free expression
```

Comment:
Is there any more side effect to support other than COMMA? If not, please remove the TODO comment becasue this PR will support COMMA side-effect.

Reply:
Removed